### ISSUE TITLE
Link share label input description missing

### ISSUE THREAD

**Issue by wiswedel** [nextcloud/server#23569]
The share label input place holder is not showing:

### Expected behaviour

![image](https://user-images.githubusercontent.com/1804221/96445662-92680780-1210-11eb-8899-af196d4249d7.png)

### Actual behaviour

![image](https://user-images.githubusercontent.com/1804221/96445057-934c6980-120f-11eb-94ee-4e875cd0ec7b.png)

### Server configuration

Server version 20.0.0.9

**Comment by wiswedel** [comment]
should have been fixed but apparently isn't: https://github.com/nextcloud/server/pull/21357/files#r453114071

**Comment by wiswedel** [comment]
cc  @gary-kim

**Comment by gary-kim** [comment]
That's strange. I remember fixing that. Huh, will take a look
